scene/resources/camera_effects.cpp

```diff
@@ -129,6 +135,7 @@ bool CameraEffects::is_override_exposure_enabled() const {
 void CameraEffects::set_override_exposure(float p_exposure) {
 	override_exposure = p_exposure;
 	_update_override_exposure();
+	notify_property_list_changed();
```

Comment:
I wonder if we couldn't call `notify_property_list_changed()` from `_validate_property` instead, only whenever something *should* change?

Reply:
Or maybe more like, why is it called here for the float property? Isn't it sufficient to update the inspector only when boolean "enabled" properties are toggled?

Reply:
Oops! These are not supposed to be there, sorry for that!